guides/doc-Release_notes/topics/katello.adoc

```diff
@@ -4,6 +4,20 @@
 [id="katello-headline-features"]
 == Headline Features
 
+For the full list of changes, see the https://github.com/Katello/katello/blob/KATELLO-{KatelloVersion}/CHANGELOG.md[Changelog].
+
+* Content view versions may now be removed from a content view in bulk via the UI.
+* A new SmartProxy Content download-policy `Streamed` has been added. This will act like a forwarding-proxy. All packages requested will be requested from the main server. This is useful if the SmartProxy does not have enough disk-space to hold all package-contents.
+* A new option to export/import just a repository.
+* Alternate Content Sources (ACSs) are now available for testing.  ACSs are alternative locations to download yum and file content from during syncing.  Enable ACSs in the UI via the "Show Experimental Labs" setting.
+* Work on the new host details page has progressed, specifically the packages and errata overview pages:
+- An Installed Products card has been added to the overview.
+- New Host collections overview card showing the associated host collections.
```

Comment:
The first "Host" probably doesn't need to be capitalized.

Reply:
```suggestion
- New host collections overview card showing the associated host collections.
```